bin/test-data/genesis/demo-tests/bitcoin-regtest/evm.json

```diff
@@ -8,7 +8,9 @@
+      "code_hash": "0xdf70b23b75d7bc725a8c0f8964fe81940ad5e223996b4049a8734e79fe47fe84",
```

Comment:
we've just removed code_hash and nonce fields form genesis files. please remove them

(see #363 #360)